src/cf-vars.less

```diff
@@ -44,15 +47,18 @@
     - cf-core
 */
 
-// Sizing variables 
+// Sizing variables
 
 @base-font-size-px:             16px;
 @base-line-height-px:           22px;
 @base-line-height:              unit(@base-line-height-px / @base-font-size-px);
 @mobile-max:                    599px;
 @tablet-min:                    600px;
+@tablet-max:                    800px;
+@desktop-min:                   801px;
+@wall-min:                      1200px;
```

Comment:
Ahhh, now I see what you were going for, @KimberlyMunoz. I don't know that I have a better suggestion, short of moving to the contextual naming scheme that @jimmynotjim suggested.